app/assets/stylesheets/customsdecexfrontend-app.scss

```diff
@@ -137,4 +137,8 @@ button:focus {
 $govuk-assets-path: '/customs-declare-exports/assets/';
 $govuk-images-path: "/customs-declare-exports/assets/lib/govuk-frontend/govuk/assets/images/";
 $govuk-fonts-path: "/customs-declare-exports/assets/lib/govuk-frontend/govuk/assets/fonts/";
-@import 'lib/govuk-frontend/govuk/all';
\ No newline at end of file
+@import 'lib/govuk-frontend/govuk/all';
+
+.details-font {
```

Comment:
This is not the correct place to but these CSS classes.
Also why do we require this?

We should not care about GDS alignment at this stage.

Reply:
Because of Font size of Details Menu Item is very small and i made it according to GDS font size 

Is it Ok ?

Reply:
![image](https://user-images.githubusercontent.com/12067582/75044060-e58a8e80-54b8-11ea-9b91-02036d8fc123.png)

Reply:
This must be in the different file. You can create a css file and add this file here.

Just realised that above we have some styles...

All styles that are above are in incorrect place :( We must extract it too
@PauloPortugal ^^

Reply:
Root cause is root element i.e HTML css
html {
    font-size: 62.5%;
}
that is coming from govuk-template.css
we should not define font-size at this level beacuse we are define some font-size in "rem" units